.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,49 @@
+name: Publish Python 🐍 distribution 📦 to PyPI
+
+on:
+  release:
+    types: [created]
```

Comment:
In the workflow I have used, I have "published" here instead of "created". 

I am not sure which one is safer to use, [here](https://stackoverflow.com/a/61066906) someone suggests that releases that are automatically generated might go directly to the "published" state without passing through the "created" one (thus "published" should be better suited here). However this is probably some sort of edge case, so feel free to leave or modify as you see fit!

Reply:
Thanks for the suggestion, I adopted it :-)